resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml

```diff
@@ -112,7 +112,7 @@ spec:
               name: router-keystore-vol
             - mountPath: /home/wso2/security/truststore
               name: router-truststore-vol
-          image: wso2/choreo-connect-router:0.9.0.2
+          image: wso2/choreo-connect-router:0.9.0
```

Comment:
These versions should be snapshot

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2000/commits/24871c5ba5b2c1fba5acf2c97ef707f9fd591787